projects/ebpf/ebpf.build/Dockerfile

```diff
@@ -37,8 +43,8 @@ RUN cat elfutils-$ELFUTILS_VERSION.tar.bz2 | tar xjf - && \
   patch -p1 < ../100-musl-compat.patch && \
   patch -p0 < ../decl.patch && \
   patch -p0 < ../intl.patch
-ENV BCC_COMMIT=d4fc95d92ec9bace9bd607dfd1833e9e06457486
+ENV BCC_COMMIT=6972806729da00ecda0235abac61d66c8fad7fad
```

Comment:
might be worth in a comment to say this is `v0.6.0` or better, use the release tag instead of the commit